HANDLING-REPORTS.md

```diff
@@ -0,0 +1,37 @@
+# Handling reports and escalation
+This document describes the proposed process for handling reports and escalation, along with relevant roles in the OpenJS Foundation.
+
+Reporting to the OpenJS Foundation either concerns a matter with the Foundation or is considered an escalation where insufficient resolution was provided by a project.
+
+## Channels
+* Email: report@openjsf.org
+  * This email address is for aliasing (like tsc@nodejs.org) to roles in handling reports and escalation.
+  *  We can use groups.io
+* Form: It would be easier on the reportee to have a form template to help them structure their report. The form submission could trigger an email
+* Repo: openjs/moderation (not provisioned)
+  * A form submission could open an issue in the repo, to allow folks to discuss in context
```

Comment:
As mentioned [here](https://github.com/openjs-foundation/cross-project-council/issues/265#issuecomment-513785281), it would probably not be appropriate to host sensitive personal information on GitHub without due consideration and review. It might be fine, but it would be Really Good to make sure first, possibly including legal consultation.